README.md

```diff
@@ -5,7 +5,13 @@ To run this demo,
 - Clone this Repo
 - Open [meeting-info.js](./meeting-info.js), read through the comments and update required information.
 - Once done, do `yarn start`
+
```

Comment:
To view how the readme looks like after these changes - https://github.com/Shreyas281299/webex-js-sdk-meeting-demo/tree/update-docs